js/animation/fx.d.ts

```diff
@@ -1,10 +1,13 @@
-import {
-    DxElement
-} from '../core/element';
+import { DxElement } from '../core/element';
+import { DxPromise } from '../core/utils/deferred';
+import { positionConfig } from './position';
 
-import {
-    DxPromise
-} from '../core/utils/deferred';
+export type animationState = string | number | 
```

Comment:
types are Upper-cased
```suggestion
export type AnimationState = string | number | 
```